### ISSUE TITLE
[Shaman][Frostband Weapon]

### ISSUE THREAD

**Issue by Healweed** [Atlantiss/NetherwingBugtracker#6623]
**Description**: 
There are SOOOO many things wrong with this ability that's so fucked up. It's not scaling as it should properly with shaman buffs/talents and on most of times it's getting Resisted. (resistances on this server are completly fucked up on overall )

I played with it on both pve and pvp and the dmg is VERY much the same.

**Current behaviour**:
**Currently the problem on this ability are the following**:

**1.** Not scaling with [Unleashed Rage](https://tbc-twinhead.twinstar.cz/?spell=30811) neither with any active attack power buffs (Bloodlust Brooch, Berserker's Call, hyjal ring etc etc)
 
**2.**  Frostband frost dmg  not getting 10% of shaman spell power ( this includes the talent [Mental Quickness](https://tbc-twinhead.twinstar.cz/?spell=30814) in which 3/3 30% of the shaman attack power is converted into spell power).
See [Wowwiki](https://wowwiki.fandom.com/wiki/Frostbrand_Weapon) note :

> Frostbrand weapon receives 10% of your spell damage equipment. For example, if you have 200 spell damage on your equipment, each Frostbrand proc will deal 20 additional damage.

 
**3.**  People should **NEVER** resist the frost dmg neither the slow itself from frostband(unless if the target is immune by any means or is immune to frost dmg). Unlike castable spells such as Frostbolt fireball and etc, Frostband is an imbue and like other imbues in the game when dealing melee dmg, it always deal the full dmg **AND** will **NEVER** be resisted at all!

**4.** ~~I have to test out of the talent [Elemental Weapons](https://tbc-twinhead.twinstar.cz/?spell=29080) to see if its benefiting or not(will edit this once i confirmed)~~ This is working and scaling well without no issues.

**Expected behaviour**: 

**1.** Frostband should always benefit 10% of the shaman's spellpower (including those from talents/active buffs)
 
**2.** Frostband should never be resisted by any means, unless the target is immune for some reason(pally divine shield, mage iceblock, rogue CoS), or against mobs that are perma immune to frost dmg (i.e of water elementals)

**Server Revision**:
3252

**Comment by Tereneckla** [comment]
1.
![image](https://user-images.githubusercontent.com/50233983/74101146-f214fb80-4b36-11ea-86bc-e8b05134c09d.png)
![image](https://user-images.githubusercontent.com/50233983/74101129-bda13f80-4b36-11ea-89fb-3c085d77bea7.png)
It is scaling.


2. Where do you get the information from that imbues can't be resisted?
Rogue poisons are also imbues and they certainly can be.

**Comment by Healweed** [comment]
**1**)Uhm..Idk how tested it out but it's sure as hell not even scaling properly. The only thing that's scaling on is based on weapon speed (meaning the more a weapon speed is slower the more damage it deals) and it's also scaling with Elemental Totems talent. And as i stated before, it's NOT scaling with any active buffs and mental quickness (Strength potion, Trinket, Unleashed Rage etc etc)

tl;dr : Imbues are not normalized and they scale according to weapon speed(and other factors i explained)

**Without Unleashed Rage:**
![nour](https://user-images.githubusercontent.com/52166185/74104688-0a911000-4b4f-11ea-885f-a47beee694fc.png)

**With Unleashed Rage active**
![WoWScrnShot_020920_150855](https://user-images.githubusercontent.com/52166185/74104691-1086f100-4b4f-11ea-9da6-0c7ff02208bf.jpg)

Ironically you can even see the Hyjal rep ring buff fading b4 i took the screenshot

**2)** I don't know about rogue's poison(and that's a different convo and different set of imbues) but  Shaman imbues are **never** meant to be resisted. As explained b4 where fireball frostbolt etc etc, self imbues will do Full damage, scaling with 10% spellpower,(including those of perma/active/temporary buffs), weapon speed and Mental Quickness talent.

**Comment by Tereneckla** [comment]
1. Read Frostbrand weapon, read sourced about Frostbrand weapon
Its damage is not dependant on the speed of the weapon. It deals the same damage no matter how fast it is. Slower weapons will only have more procs, as it is a PPM ability. 
The base damage of Frostbrand weapon is 245 at rank 6.
Base damage with 3/3 Elemental Weapons is 245*1.15 = 281
As there is no range on the damage all your images even prove that temporary buffs apply to the damage it deals.
